app/controllers/api/base_controller.rb

```diff
@@ -1,3 +1,14 @@
 class Api::BaseController < ApplicationController
   layout nil
+
+  # Method to log login attempts
+  def log_login_attempt(user_id:, attempted_at:, successful:, ip_address:)
```

Comment:
Ensure that the 'user_id', 'attempted_at', 'successful', and 'ip_address' parameters are properly validated and sanitized to prevent potential security issues such as injection attacks.